holoviews/plotting/widgets/widgets.js

```diff
@@ -539,3 +541,88 @@ for (var k in _namespace) {
     window.HoloViews[k] = _namespace[k];
   }
 }
+
+var JS_MIME_TYPE = 'application/javascript';
+var HTML_MIME_TYPE = 'text/html';
+var EXEC_MIME_TYPE = 'application/vnd.holoviews_exec.v0+json';
+var CLASS_NAME = 'output';
+
+/**
+ * Render data to the DOM node
+ */
+function render(props, node) {
+  var script = document.createElement("script");
+  node.appendChild(script);
+}
+
+/**
+ * Handle when a new output is added
+ */
+function handleAddOutput(event, handle) {
+  var output_area = handle.output_area;
+  var output = handle.output;
+  // limit handleAddOutput to display_data with EXEC_MIME_TYPE content only
+  if ((output.output_type != "display_data") || (!output.data.hasOwnProperty(EXEC_MIME_TYPE))) {
+    return
+  }
+  var toinsert = output_area.element.find("." + CLASS_NAME.split(' ')[0]);
+  if (output.metadata[EXEC_MIME_TYPE]["id"] !== undefined) {
+    toinsert[0].firstChild.textContent = output.data[JS_MIME_TYPE];
+    output_area._hv_plot_id = output.metadata[EXEC_MIME_TYPE]["id"];
+  }
+}
+
+/**
+ * Handle when an output is cleared or removed
+ */
+
+
+function handleClearOutput(event, handle) {
+  var id = handle.cell.output_area._hv_plot_id;
+  if (id === undefined) { return; }
+  var comm = window.HoloViews.comm_manager.get_client_comm("hv-extension-comm", "hv-extension-comm", function () {});
+  if (comm !== null) {
+    comm.send({event_type: 'delete', 'id': id});
+  }
+  if ((window.Bokeh !== undefined) & (id in window.Bokeh.index)) {
+    window.Bokeh.index[id].model.document.clear();
+    delete Bokeh.index[id];
+  }
+}
+
+function register_renderer(events, OutputArea) {
```

Comment:
Bit of an inconsistent mix of snake case and camel case going on here...

Reply:
Intentional, ``OutputArea`` is the name of a class.

Reply:
I'm talking about the functions in this file.

Reply:
Ah, thanks for clarifying, that was confusing.

Reply:
Fixed.